"make install" does not work

### Issue checklist

- [x] I have read the [README](https://github.com/NOAA-GSL/SENA-yppm/blob/develop/README.md) and [ref/README](https://github.com/NOAA-GSL/SENA-yppm/blob/develop/ref/README.md), including the Troubleshooting Guide.
- [x] I have reviewed existing issues to ensure this has not already been reported.

### Describe the bug

make install not working

### Recipe to reproduce

Steps to reproduce the behavior:
```
$ make install
make: *** No rule to make target `install'.  Stop.
```
### Expected behavior

When "make install" worked, the executable, yppm, would be placed in ./ref/exe

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -53,6 +53,7 @@ jobs:
           cmake -DCMAKE_BUILD_TYPE=debug ..
           make
           ctest
+          make install
           
       # Test release mode 
       - name: Test yppm Release
@@ -66,7 +67,8 @@ jobs:
           cmake -DCMAKE_BUILD_TYPE=release ..
           make
           ctest       
-       
+          make install
+
   ubuntu_build:
     name: Ubuntu Build
     # Run on ubuntu-latest
@@ -109,6 +111,8 @@ jobs:
           cmake -DCMAKE_BUILD_TYPE=debug ..
           make
           ctest
+          make install
+
       # Test release mode
       - name: Build yppm release
         run: |
@@ -121,3 +125,4 @@ jobs:
           cmake -DCMAKE_BUILD_TYPE=release ..
           make
           ctest
+          make install

--- a/ref/src/CMakeLists.txt
+++ b/ref/src/CMakeLists.txt
@@ -17,3 +17,5 @@ if(OpenMP_FOUND)
     target_link_libraries(${PROJECT_NAME} PUBLIC OpenMP::OpenMP_C)
 endif()
 
+install(TARGETS ${PROJECT_NAME}
+        DESTINATION ${PROJECT_SOURCE_DIR}/exe)

--- a/ref/test/CMakeLists.txt
+++ b/ref/test/CMakeLists.txt
@@ -115,3 +115,6 @@ add_test(NAME compare_0.0.12_30
 add_test(NAME InterpolateTest 
          COMMAND bash -c "set -o pipefail; ../test/interpolateTest 2>&1 | tee test_output/interpolateTest.out")
 
+# Test install
+add_test(NAME install
+         COMMAND make install)

